packages/fuels/tests/harness.rs

```diff
@@ -3761,9 +3761,9 @@ async fn test_script_interface() -> Result<(), Error> {
     let script = vec![
         Opcode::gtf(0x10, 0x00, GTFArgs::ScriptData),
         Opcode::ADDI(0x11, 0x10, ContractId::LEN as u16),
-        Opcode::LW(0x11, 0x11, 0),
-        Opcode::ADDI(0x12, 0x11, WORD_SIZE as u16),
-        Opcode::TR(0x10, 0x11, 0x12),
+        Opcode::LW(0x12, 0x11, 0),
+        Opcode::ADDI(0x13, 0x11, WORD_SIZE as u16),
+        Opcode::TR(0x10, 0x12, 0x13),
```

Comment:
Oof, nice find!

Reply:
Oh come we didn't catch in the tests ?

Reply:
Yup, the previous test was just weak. My bad.